workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -1,20 +1,31 @@
+using System.Collections.Generic;
```

Comment:
This file is being rather massive, is it possible to split some of the code out? For example, the bit dealing with the XML documents.

Reply:
the one with the xml document is specific to iOS xctestrun. can't really be used for any other kind of XML documents. it's because they use a different format from most xml docs. We can split it into AndroidLaunchMenu and iOSLaunchMenu?